locale/pt_BR/admin.xml

```diff
@@ -13,33 +13,50 @@
+	<message key="admin.auth.enableSyncProfiles">Habilitar sincronização de perfil de usuários (caso seja compatível com o plugin de autenticação!). Os perfis dos usuários serão atualizados com o conteúdo da fonte remota assim que o usuário acessar o sistema, e alterações no perfil (incluindo alteração de senha) realizadas dentro do sistema atualizarão a fonte remota. Caso esta opção não esteja habilitada, os perfis no sistema serão mantidos separados da fonte remota.</message>
+	<message key="admin.auth.enableSyncPasswords">Habilitar alteração de senha de usuário (caso seja compatível com o plugin de autenticação!). Habilitar esta opção permite ao usuário modificar a senha do sistema e fazer uso da opção de recuperação de senha para criar uma nova senha. Estas funcionalidades não estarão disponíveis aos usuários com esta fonte de autenticação, caso esta opção não seja habilitada.</message>
+	<message key="admin.auth.enableCreateUsers">Habilitar criação de usuários (caso seja compatível com o plugin de autenticação!). Usuários criados no sistema serão automaticamente incluídos na fonte de autenticação remota, caso não existam ainda. Inclusive, usuários criados através do cadastro de usuários também serão incluídos automaticamente na fonte remota, caso esta seja a fonte de autenticação padrão.</message>
 	<message key="admin.systemVersion">Versão do OJS</message>
 	<message key="admin.systemConfiguration">Configuração do OJS</message>
 	<message key="admin.systemConfigurationDescription"><![CDATA[Configurações do OJS no arquivo <tt>config.inc.php</tt>.]]></message>
+	<message key="admin.journals.createInstructions">Seu cadastro como Gerente da Revista será automático. Após a criação de uma revista, acesse-a como gerente e continue com as configurações da revista e cadastro de usuários.</message>	
 	<message key="admin.journals.journalSettings">Configurações da revista</message>
 	<message key="admin.journals.noneCreated">Nenhuma revista foi criada.</message>
 	<message key="admin.contexts.confirmDelete">Deseja realmente excluir permanentemente esta revista e seu conteúdo?</message>
 	<message key="admin.contexts.create">Criar revista</message>
-	<message key="admin.journals.createInstructions">Seu cadastro como Gerente da Revista será automático. Após a criação de uma revista, acesse-a como gerente e continue com as configurações da revista e cadastro de usuários.</message>
 	<message key="admin.journals.urlWillBe">Geralmente é usado um acrônimo que identifica o periódico. A URL da revista será: {$sampleUrl}</message>
+<message key="admin.contexts.form.titleRequired">O título é obrigatório</message>
```

Comment:
@aaabulafiaaa, please make sure your editor is set up to indent with tabs rather than spaces...